### DISCUSSION TITLE
FBA Inventory API, getInventorySummaries - Rate throttling

### DISCUSSION THREAD

**Discussion by pellet8** [abuzuhri/Amazon-SP-API-CSharp#289] [General]
According to SP-API doc, https://developer-docs.amazon.com/sp-api/docs/fbainventory-api-v1-reference, 
GET /fba/inventory/v1/summaries is rate limited to 2 requests per second. However, the GetInventorySummaries call will throw a QuotaExceeded error. Please review.

**Comment by pellet8** [comment]
Nevermind. The issue seems to be resolved in version 1.4.23

**Comment by abuzuhri** [comment]
Thanks
